GVFS/GVFS.UnitTests.Windows/Windows/Mock/MockProductUpgrader.cs

```diff
@@ -32,7 +32,9 @@ public enum ActionType
             GitDownload = 0x4,
             GVFSDownload = 0x8,
             GitInstall = 0x10,
-            GVFSInstall = 0x20
+            GVFSInstall = 0x20,
+            GVFSCleanup = 0x40,
+            GitCleanup = 0x80

```

Comment:
Nit: a trailing comma will make future diffs cleaner (e.g. see the `GVFSInstall` line, that would have been white if it had a trailing comma in the original change)